provider/postgis/postgis.go

```diff
@@ -315,44 +314,33 @@ func BuildDBConfig(opts *DBConfigOptions) (*pgxpool.Config, error) {
 		return nil, err
 	}
 
-	dbconfig.ConnConfig.LogLevel = pgx.LogLevelWarn
```

Comment:
Note to self. Has this log level config been moved to somewhere else?

Reply:
> Note to self. Has this log level config been moved to somewhere else?

yes it did. instead of a logger within pgx, logging is extracted to be in the pgx tracelog package. I'll have to take a look on how to make that work for us later on.

Reply:
Hm, i'd have to build a log adapter for the tracelog package of pgx for our custom logger. 🤔 

Something like so:

```go
        // this would be an instance of slog logger
	logAdapter := log.NewLoggerAdapter(logger)
	tracer := &tracelog.TraceLog{
		Logger:   logAdapter,
		LogLevel: tracelog.LogLevelNone,
	}
	if <pg debug enabled> {
		tracer.LogLevel = tracelog.LogLevelDebug
	}
	conf.ConnConfig.Tracer = tracer
```

there's readily available adapters for slog already. I don't think it make sense to build a custom logadapter for our custom logger if we agree that slog should be come the default logger and we drop str logger and zap. wdyt?

Reply:
I agree that we should move towards adopting `slog`, but I'm not sure if we should land this PR without a logger in place. One option is that we could shift over to the slog refactor, land that, and then circle back to this PR. If we don't want to do that, I think implementing the custom log adapter would be the way to go, even if it's temporary.

Reply:
There is a logger, tegola logger remains untouched by this. PGX however does not come with logger any longer, but expects the user to pass it if required.
I will add a very simple adapter that reflects the previous behaviour of:

```
dbconfig.ConnConfig.LogLevel = pgx.LogLevelWarn
```

Which is currently logging anything that pgx considers as warning.